### ISSUE TITLE
Simple

### ISSUE THREAD

**Issue by HeinekenBottle** [HeinekenBottle/OXIDE#3]
@claude 

Design: small, strict, fast. Keep only what trains “start-at-10” on clean windows.

1) Repo layout
OXIDE/
  corefeed/                # raw bars (DataBento)
  datasets/
    canonical_41_min.feather
    split_v1.json
  feedback/
    binary/ batch_*.csv
    soft/   soft_*.jsonl
  models/OXIDE-Exp-Start-T1/1.1.0/
  runs/
  src/
    audit/{collect_unify.py,build_windows.py,report.py}
    core/preprocess.py      # normalize_ohlc, build_minimal_features
    data/dataset_41.py      # 41-bar loader, masks
    training/train_t1.py    # phases S,B
    notebooks/generator.py  # 4-cell correction lab
  scripts/
    audit/run_agent_audit.sh
    make_split.py
  memory/constitution.md
  templates/{tasks-template.md}
  .claude/commands/{agent-audit.md,fast-spec.md}
  Makefile

2) Data schema (single CSV)

corrections_unified.csv

event_id,chart_num,symbol,session_id,ts_window_left,ts_corrected?,
binary∈{0,1},soft∈[0,1|null],confidence?,source

3) Canonical slice (single rule)

Re-slice from raw for each correction:

window = [ts_corr−10m … ts_corr+30m]; len==41; index10==ts_corr

Drop if len≠41. No cached slices.

4) Minimal features (price-only)

After slicing:

close_norm, hl_range, ret_1m, cum_ret, pos_in_window

No syn_vol. No ITF/HTF.

5) Split

Deterministic 60/20/20 by session → split_v1.json. Store split in dataset.

6) Training objective

Two heads, no 41-class:

loss = 1.0*CE_binary + 0.5*BCEWithLogits(soft)  # mask soft when missing

7) Training order

Phase S (soft only) → Phase B (binary+soft):

make train_soft
make train_bin

8) Makefile (one-touch)
.PHONY: audit build split train_soft train_bin sample notebook

audit:
	python -m src.audit.collect_unify --bin 'feedback/binary/*.csv' --soft 'feedback/soft/*.jsonl' --out reports/audit
	python -m src.audit.build_windows --in reports/audit/corrections_unified.csv --bars corefeed/raw/nq_v0 --out datasets/canonical_41.parquet
	python -m src.audit.report --in reports/audit --out reports/audit/audit_report.md

build:
	python -m src.core.preprocess --in datasets/canonical_41.parquet --out datasets/canonical_41_min.feather

split:
	python scripts/make_split.py --in datasets/canonical_41_min.feather --out datasets/split_v1.json

train_soft:
	python scripts/training/train_t1.py --dataset datasets/canonical_41_min.feather --split datasets/split_v1.json --soft-weight 1.0 --binary-weight 0.0 --run_id t1_soft

train_bin:
	python scripts/training/train_t1.py --resume runs/t1_soft --soft-weight 0.5 --binary-weight 1.0 --replay --run_id t1_bin

sample:
	python -m src.training.sample --runs runs/t1_bin --out runs/t1_bin/samples_100.csv

notebook:
	python -m src.notebooks.generator --samples runs/t1_bin/samples_100.csv --out notebooks/T1_corrections_100.ipynb

9) Guardrails (hard asserts)

Trainer init:

assert all(h.out_dim in (1,2) for h in model.heads)

Dataloader:

assert len(win)==41 and win.index[10]==ts_corr

CI smoke (Make ci_fast):

import check

build 5 windows

20-step train_soft, 20-step train_bin

10) Acceptance checks (promotion gate)

Promote if:

(AUC_val ≥ baseline+0.01) OR (AUC_val ≥ baseline−0.01 AND ECE_val ≤ baseline)

Artifacts:

runs/t1_bin/{checkpoint.pt,metrics.json,config.json}
models/OXIDE-Exp-Start-T1/1.1.0/checkpoint.pt

11) Claude agents (two only)

/agent-audit (already built) → outputs unified CSV + canonical_41 + audit.

/directive reminder (41-bar semantics, no global offsets, no 41-class).

12) What gets deleted

Old V7–V13 loaders and any 41-class timing heads.

Synthetic data generators used for training.

Redundant notebooks. Keep generator only.

Any cached slices that predate ts_corr re-anchoring.

13) Daily workflow
make audit build split train_soft train_bin sample notebook

Review 100-sample notebook. Add new corrections. Repeat.

**Comment by HeinekenBottle** [comment]
@claude Is there any else to reduce 80/20 style?
